include/internal/libspdm_common_lib.h

```diff
@@ -1132,4 +1132,15 @@ void libspdm_get_receiver_buffer (
     void **receiver_buffer,
     size_t *receiver_buffer_size);
 
+/**
+ * Get the certificate slot mask
+ *
+ * @param[in]   context              A pointer to the SPDM context.
+ * @param[out]  slot_mask            address of get slot mask
+ *
+ **/
+void libspdm_get_cert_slot_mask(
```

Comment:
No need to return via parameter. please follow my suggestion:

```
uint8_t libspdm_get_cert_slot_mask (libspdm_context_t *spdm_context);
```

Reply:
I have changed the code. Thanks.